### PR TITLE
Update build status on release-1.4

### PR DESCRIPTION
**Pull request by mboukhalfa**
Update build status after migrating tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Metal3 IP Address Manager for Cluster API Provider Metal3
 
-[![Ubuntu daily 1.4 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-4_integration_test_ubuntu&subject=Ubuntu%20daily%201.4)](https://jenkins.nordix.org/view/Metal3/job/metal3_daily_release-1-4_integration_test_ubuntu/)
-[![CentOS daily 1.4 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-4_integration_test_centos&subject=CentOS%20daily%201.4)](https://jenkins.nordix.org/view/Metal3/job/metal3_daily_release-1-4_integration_test_centos/)
+[![Ubuntu E2E Integration 1.4 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_periodic_release-1-4_integration_test_ubuntu&subject=Ubuntu%20e2e%20integration%201.4)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_periodic_release-1-4_integration_test_ubuntu/)
```
Comment (tuminoid) [inline README.md:3]: It says "not run", and when clicked "not found", so the link is wrong.

**Comment by mboukhalfa** [comment]
Closed in favor of https://github.com/metal3-io/ip-address-manager/pull/504 since release-1.4 is not longer supported after releasing-1.7

**Pull request closed without merging**